package.json

```diff
@@ -15,7 +15,9 @@
     "underscore": "1.12.1",
     "hosted-git-info": "3.0.8",
     "url-parse": "1.5.1",
-    "postcss": "8.2.10"
+    "postcss": "8.2.10",
+    "browserslist": "^4.16.6",
```

Comment:
we pin dependencies in simorgh so you you'll need to make this `"browserslist": "4.16.6"`

Reply:
Thanks for the heads-up @jroebu14. I have pinned it :)